channels/dz.m3u

```diff
@@ -1,54 +1,26 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="ALANISTV.dz" tvg-name="AL ANIS TV" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/G60iJen.png" group-title="",AL ANIS TV
 https://cdn02.hta.dz/abr_htatv/El_Fhama_TV/playlist.m3u8
-#EXTINF:-1 tvg-id="AlgerieTV4.dz" tvg-name="Algerie TV4" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/n8httHt.png" group-title="",Algerie TV4
-https://cdn02.hta.dz/abr_htatv/TV_4/playlist.m3u8
```

Comment:
All of those links simply had a hard time. Not 24/7.

Reply:
Currently streaming